stacks/census2lazyloads/templates.html

```diff
@@ -0,0 +1,77 @@
+<!--
+Templates allow you to change how html is rendered into a juicebox stack. See the templates guide for more details.
```

Comment:
I think you can delete this template.html file, because there are no slices in this stack that use any of the templates defined in it.